vspace/src/main/java/edu/asu/diging/vspace/web/staff/ModuleController.java

```diff
@@ -5,25 +5,29 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
-
 import edu.asu.diging.vspace.core.model.IModule;
-import edu.asu.diging.vspace.core.model.impl.ModuleStatus;
+import edu.asu.diging.vspace.core.services.IModuleLinkManager;
 import edu.asu.diging.vspace.core.services.IModuleManager;
 
 @Controller
 public class ModuleController {
 
     public static final String STAFF_MODULE_PATH = "/staff/module/";
+
     @Autowired
     private IModuleManager moduleManager;
 
+    @Autowired
+    private IModuleLinkManager moduleLink;
```

Comment:
This is very confusing. The object is not a module link but a module link manager. Right now line 30 implies that a module link has a method to get a space list from a module id.

Reply:
Changed the object name to moduleLinkManager